corona-warn-app/cwa-website
# Correct sitemap links to 2.20 blog

This PR resolves issue #2765 "Bad links to latest blog in sitemap". It corrects the bad links in [src/data/sitemap.json](https://github.com/corona-warn-app/cwa-website/blob/master/src/data/sitemap.json) and [src/data/sitemap_de.json](https://github.com/corona-warn-app/cwa-website/blob/master/src/data/sitemap_de.json).

## Verification

The following actions should show no 404 error and should display the corresponding blog:

- Open https://www.coronawarn.app/en/sitemap/ and click on first blog
- Open https://www.coronawarn.app/de/sitemap/ and click on first blog
